dashboard/app/models/net_sim.rb

```diff
@@ -95,9 +95,9 @@ def self.message_granularity_options
 
   def self.create_from_level_builder(params, level_params)
     create!(level_params.merge(
-              user: params[:user],
-              game: Game.netsim,
-              level_num: 'custom'
+      user: params[:user],
+      game: Game.netsim,
+      level_num: 'custom'
             ))
```

Comment:
Yep!  Rubocop just merged a fix for the `Style/ClosingParenthesisIndentation` cop: https://github.com/bbatsov/rubocop/issues/1758

I can hand-fix occurrences in this PR, then enable the cop once it ships.